can-ajax.js

```diff
@@ -227,11 +228,15 @@ function ajax(o) {
 		xhr.setRequestHeader("X-Requested-With", "XMLHttpRequest");
 	}
 
-        if (o.xhrFields) {
-            for (var f in o.xhrFields) {
-                xhr[f] = o.xhrFields[f];
-            }
-        }
+	if(o.beforeSend){
+		o.beforeSend.call( null, xhr, o );
```

Comment:
is `null` important here?  Might folks assume `this` works against the options (`o`)?